test/object_store/test_basic.py

```diff
@@ -76,19 +77,25 @@ def kill_with_dir(old_pid, run_dir):
     os.unlink(scylla_link)
 
 
+class Cluster:
+    def __init__(self, cql, ip: str):
+        self.cql = cql
+        self.ip = ip
+        self.api = ScyllaRESTAPIClient()
+
```

Comment:
after a second thought, if we want to improve the usability of `self.api` in future, we could have something like:
```py
class ScyllaRestClient:
    """delegate the calls to the wrapped client"""
    def __init__(self, client: ScyllaRESTAPIClient, server_ip: str) -> None:
        self.ip_addr = server_ip
        self.client = client

    def __getattr__(self, name) -> Callable:
        method = getattr(self.client, name)
        return partial(method, self.ip_addr)
```

Reply:
I believe it's better to fix ScyllaRESTAPIClient to accept optional "default" IP address and make all its method use it if not provided by the argument

Reply:
In other tests the api client connects to more than one server. I'd wrap it in cases like this one.

Or it could be an optional argument in both the constructor and the request. But it would be quite verbose and will need checks.